### DISCUSSION TITLE
Make visibility and business optional on PUT /api/v1/status

### DISCUSSION THREAD

**Discussion by jeriox** [Traewelling/traewelling#2930] [Ideas / Feature Requests]
I keep forgetting to select the correct event for my checkins. As there is currently no UI to edit the event after checking in, I updated my checkin manually via the ``PUT /status`` API route by providing the corresponding event ID. However, visibility and business are mandatory fields for this endpoint (cf https://github.com/Traewelling/traewelling/blob/47d9ab8737f0d5de4819734cebba177b4a350fbb/app/Http/Controllers/API/v1/StatusController.php#L399). It would be nice to make them optional in case they should not be updated (as in my example).

**Comment by fzakfeld** [comment]
This would be considered a PATCH request then
